defaults/main.yml

```diff
@@ -41,11 +41,11 @@ pdns_rec_install_epel: True
 # to be created by the PowerDNS Recursor package or by an other role.
 # If you change these variables, make sure to create the user and groups before
 # applying this role
-pdns_rec_user: pdns
-pdns_rec_group: pdns
+pdns_rec_user: "{% if ansible_os_family in ['RedHat'] %}pdns-recursor{% else %}pdns{% endif %}"
```

Comment:
This looks good to me. :)

I think we could achieve the same result keeping the `vars/{{ ansible_os_family }}.yml`and having in `defaults/main.yml`

```
pdns_rec_user: "{{ default_pdns_rec_user }}"
pdns_rec_group: "{{ default_pdns_rec_group }}"
```

in `vars/main.yml`

```
default_pdns_rec_user: pdns
default_pdns_rec_group: pdns
```
and finally in `vars/RedHat.yml`

```
default_pdns_rec_user: pdns-recursor
default_pdns_rec_group: pdns-recursor
```